### PR TITLE
Update namespaces.md

### PR DESCRIPTION
**Pull request by stewart-yu**
fix bad json link

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6876)

### CODE REVIEW

File: docs/tasks/administer-cluster/namespaces-walkthrough.md [hunk 1]
```diff
@@ -53,14 +53,14 @@ One pattern this organization could follow is to partition the Kubernetes cluste
...
-Use the file [`namespace-dev.json`](/docs/admin/namespaces/namespace-dev.json) which describes a development namespace:
+Use the file [`namespace-dev.json`](/docs/tasks/administer-cluster/namespace-dev.json) which describes a development namespace:
 
 {% include code.html language="json" file="namespace-dev.json" ghlink="/docs/tasks/administer-cluster/namespace-dev.json" %}
 
 Create the development namespace using kubectl.
 
 ```shell
-$ kubectl create -f docs/admin/namespaces/namespace-dev.json
+$ kubectl create -f /docs/tasks/administer-cluster/namespace-dev.json
```
Comment (zhangxiaoyu-zidif) [inline docs/tasks/administer-cluster/namespaces-walkthrough.md]: here we do not need add slash

**Comment by zhangxiaoyu-zidif** [comment]
small nit ，otherwise lgtm

File: docs/tasks/administer-cluster/namespaces-walkthrough.md (hunk 1 above)
Comment (stewart-yu) [reply docs/tasks/administer-cluster/namespaces-walkthrough.md]: Sorry, copy mistakes. Done, thanks.

Comment (zhangxiaoyu-zidif) [reply docs/tasks/administer-cluster/namespaces-walkthrough.md]: ok， let's wait until CI pass

File: docs/tasks/administer-cluster/namespaces-walkthrough.md [hunk 2]
```diff
@@ -53,14 +53,14 @@ One pattern this organization could follow is to partition the Kubernetes cluste
...
-Use the file [`namespace-dev.json`](/docs/admin/namespaces/namespace-dev.json) which describes a development namespace:
+Use the file [`namespace-dev.json`](/docs/tasks/administer-cluster/namespace-dev.json) which describes a development namespace:
 
 {% include code.html language="json" file="namespace-dev.json" ghlink="/docs/tasks/administer-cluster/namespace-dev.json" %}
 
 Create the development namespace using kubectl.
 
 ```shell
-$ kubectl create -f docs/admin/namespaces/namespace-dev.json
+$ kubectl create -f docs/tasks/administer-cluster/namespace-dev.json
```
Comment (tengqm) [inline docs/tasks/administer-cluster/namespaces-walkthrough.md]: I'd suggest we provide sample instructions in one of the following two ways:

- If the input is provided in a `{% include ... %}` block already (the exact case here), we avoid using relative paths in the commands. In this case, we assume the user has already downloaded the file to his/her local machine. I.e.

  ``` shell
  $ kubectl create -f namespace-dev.json
  ```
- We can also provide the _full_ URL to a file in sample commands, i.e.:

  ```shell
  $ kubectl create -f https://k8s.io/docs/tasks/administer-cluster/namespace-dev.json
  ```

But we should not use relative paths in documentation. It means the user has to clone the website to try things out and it is confusing.

Comment (stewart-yu) [reply docs/tasks/administer-cluster/namespaces-walkthrough.md]: Yeah, you are right, some relative url  also confuse me

Comment (zhangxiaoyu-zidif) [reply docs/tasks/administer-cluster/namespaces-walkthrough.md]: right, I ignored that, thanks @tengqm

**Comment by stewart-yu** [comment]
Any question about this PR? @tengqm

**Comment by tengqm** [comment]
@stewart-yu still looking at CI progress.

**Pull request merged**